src/more-infos/more-info-cover.html

```diff
@@ -10,20 +10,24 @@
   <template>
     <style is="custom-style" include="iron-flex"></style>
     <style>
-      .current_position, .current_tilt_position {
+      .current_position, .tilt {
         max-height: 0px;
         overflow: hidden;
       }
       .has-current_position .current_position,
-      .has-current_tilt_position .current_tilt_position {
+      .has-open_tilt .tilt,
+      .has-close_tilt .tilt,
+      .has-stop_tilt .tilt,
+      .has-set_tilt_position .tilt,
+      .has_current_tilt_position .tilt
```

Comment:
Sorry these are coming in one at at time, but should be `has-current_tilt_position` instead of `has_current_tilt_position`